proto/server/pi_server.cpp

```diff
@@ -631,13 +632,29 @@ void PIGrpcServerInitWithConfig(const char *config_text, const char *version) {
   assert(status.code() == ::google::rpc::Code::OK);
 }
 
-void PIGrpcServerRunAddrGnmi(const char *server_address, void *gnmi_service) {
+void PIGrpcServerRunV2(const char *server_address,
+                       void *gnmi_service,
+                       PIGrpcServerSSLOptions_t *ssl_options) {
   server_data = new ::pi::server::ServerData();
   server_data->server_address = std::string(server_address);
   auto &builder = server_data->builder;
+  std::shared_ptr<grpc::ServerCredentials> creds;
+  if (ssl_options == nullptr) {
+    creds = grpc::InsecureServerCredentials();
+  } else {
+    grpc::SslServerCredentialsOptions::PemKeyCertPair pkcp;
+    pkcp.private_key = (ssl_options->pem_private_key == NULL) ?
+        "" : ssl_options->pem_private_key;;
```

Comment:
Typo, double semicolon

Reply:
Thanks, feel free to open a PR to fix it or I can take care of it next time I merge a PR in this repo.